iron-meta.js

```diff
@@ -74,14 +72,25 @@ IronMeta.prototype = {
         return metaDatas[this.type][key];
       }, this);
     }
-  },
+  }
 
-  byKey: function(key) {
+  /**
+   * @param {string} key
```

Comment:
Should this match the annotations above with `{string|null|undefined}`?

Reply:
if key is not truthy you always get undefined, so it seems to make more sense for this method to require you provide a key